pulp-workflow-overview.md

```diff
@@ -121,12 +139,108 @@ This is the workflow illustrated in this example:
 2. A package is built and uploaded into the _Dev_ repository.
 3. Packages in the _Dev_ repository are then tested, for example, using integration tests.
 There might be some back and forth, which creates different repository versions.
-4. When all tests have passed, the packages are promoted to the _Test_ repository.
+4. When all tests have passed, the packages are promoted* to the _Test_ repository.
 5. Quality engineering (QE) syncs packages from the _Test_ repository. QE passes/fails certain tests.
-6. Devs provide patches that they then promote new versions to the Test Repository.
-7. When tests complete, QE promotes the packages to the _Production_ (_Prod_) repository.
+6. Devs provide patches that they then promote* new versions to the Test Repository.
+7. When tests complete, QE promotes* the packages to the _Production_ (_Prod_) repository.
 8. Members of your organization or your customers install packages from the Production repository.
 
+**Promote**
+
+In these examples, we use the term promote here to mean cloning the contents of a repository.
+For example, at which ever point you want, you can create a repository version based on the other this version.
+Here is an example in the [RPM documentation](https://docs.pulpproject.org/pulp_rpm/workflows/copy.html#basic-repository-modification-api)
+However, you might prefer to point to a repository version rather than clone a repository.
+Both options serve the same purpose.
+Many Pulp users find it useful to have separate repositories for each environment because it's easier to deal with a history of repository versions in each repository than when all of them are in one repo.
+
+Here is a truncated example containing just dev and prod.
+This workflow assumes the installation and configuration of Pulp, as well as the Pulp CLI.
+
+1. Create a remote that points to PyPI:
+
+```
+pulp python remote create --name "dev" --url "https://pypi.org/" --includes '["shelf-reader"]'
+```
+
+2. Create a remote that points to your Pulp server so that clients can packages from Pulp:
```

Comment:
"prod" get packages from "dev" (ideally QA), like the packages from external world were tested and considered safe, that's why prod don't get them from PyPI